### PR TITLE
docs: update getting started guide with initial feedback

### PR DESCRIPTION
**Pull request by avasisht23**
# Pull Request Checklist

- [ ] Did you add new tests and confirm existing tests pass? (`yarn test`)
- [ ] Did you update relevant docs? (docs are found in the `site` folder, and guidelines for updating/adding docs can be found in the [contribution guide](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md))
- [ ] Do your commits follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] Does your PR title also follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] If you have a breaking change, is it [correctly reflected in your commit message](https://www.conventionalcommits.org/en/v1.0.0/#examples)? (e.g. `feat!: breaking change`)
- [ ] Did you run lint (`yarn lint:check`) and fix any issues? (`yarn lint:write`)
- [ ] Did you follow the [contribution guidelines](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md)?

---

## PR-Codex overview
The focus of this PR is to update file extensions, remove unused imports, and refine content related to modular accounts and authentication.

### Detailed summary
- Updated file extensions in `package.json` for `.tsx` files
- Removed unused file extensions in `globals.css` import
- Refactored references to `Modular Smart Contract Account` to `Modular Account`
- Updated content related to `Magic Link Authentication` in `terms.md`

> The following files were skipped due to too many changes: `site/getting-started/overview.md`, `site/getting-started/send-uos.md`, `site/getting-started/log-in-users.md`, `site/getting-started/setup-app.md`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Review by rthomare — APPROVED** [review]

**Review by SahilAujla — DISMISSED** [review]
Looks great @avasisht23!

Only thing is, we need to add the file extensions back in imports, otherwise the linting fails. 

image

Once we've added them back, we can merge, so approving now.

**Comment by avasisht23** [comment]
Yeah we should ignore the linter for the `site/snippets` folder.

Leeds was copying those files directly from docs into his code and they were failing. Clear signal of confusion when following this doc to a tee.

File: site/getting-started/introduction.md [hunk 1]
```diff
@@ -20,20 +20,16 @@ head:
 
 # Quick start
 
-This guide will show you how to create Embedded Accounts using Account Kit, the Alchemy Signer, and the Alchemy Modular Smart Contract Account.
+This guide will show you how to create Embedded Accounts using [Account Kit](https://accountkit.alchemy.com/resources/terms.html#account-kit), the Alchemy [Signer](https://accountkit.alchemy.com/resources/terms.html#signer), and the Alchemy [Modular Smart Contract Account](https://accountkit.alchemy.com/resources/terms.html#modular-account).
```
Comment (moldy530) [inline site/getting-started/introduction.md:23]:
```suggestion
This guide will show you how to create Embedded Accounts using [Account Kit](https://accountkit.alchemy.com/resources/terms.html#account-kit), the Alchemy [Signer](https://accountkit.alchemy.com/resources/terms.html#signer), and the Alchemy [Modular Account](https://accountkit.alchemy.com/resources/terms.html#modular-account).
```

Comment (moldy530) [reply site/getting-started/introduction.md:23]: we're just calling it MA everywhere now so we should do a pass and make sure that holds in our docs as well

File: site/getting-started/introduction.md [hunk 2]
```diff
@@ -20,20 +20,16 @@ head:
...
+
+::: tip Want to test it out right away?
+Clone [this Github repo](https://github.com/alchemyplatform/embedded-accounts-quickstart) and follow the `README`.
+:::
 
 <img src="/images/getting-started/embedded-accounts-banner.png" alt="Embedded Accounts Banner" />
```
Comment (moldy530) [inline site/getting-started/introduction.md:29]: I think in the original mocks we had this at the very top no?

File: site/getting-started/setup-app.md [hunk 3]
```diff
@@ -20,12 +20,74 @@ head:
...
+
+<img src="/images/getting-started/api-key-configure.png" alt="API Key Configure" />
+
+Click the API Key button in the top right corner and copy-paste it into the `.env` file of your application as an environment variable called `ALCHEMY_API_KEY`.
+
+<img src="/images/getting-started/api-key-view.png" alt="API Key View" />
+
+### Alchemy Embedded Accounts Config
+
+The Embedded Accounts Config enables magic auth on your app's domain by configuring the Alchemy Signer, which securely stores the user's private key in a non-custodial [secure enclave](https://docs.turnkey.com/security/our-approach). It is responsible for authenticating a user via email or passkey using this config, managing a user's session, and signing messages to send UserOperations. Check out the [AlchemySigner docs](https://accountkit.alchemy.com/packages/aa-alchemy/signer/overview.html) for more details.
```
Comment (moldy530) [inline site/getting-started/setup-app.md:53]: what's `magic auth` here?

should it be `magic link auth`?

**Review by moldy530 — APPROVED** [review]
couple comments but LGTM overall

**Review by moldy530 — APPROVED** [review]

**Pull request merged**